src/Microsoft.DotNet.Interactive/Commands/KernelCommand.cs

```diff
@@ -35,11 +35,11 @@ public KernelCommand Parent
         internal set
         {
             _parent = value;
-            var currentSlip = RoutingSlip;
+            var currentSlip = RoutingSlip ?? new RoutingSlip();
```

Comment:
If `RoutingSlip` is null that's a bug. Let's not hide it by silently coalescing here.